share/functions/fish_default_key_bindings.fish

```diff
@@ -2,7 +2,9 @@
 function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fish"
 	if not set -q argv[1]
 		if test "$fish_key_bindings" != "fish_default_key_bindings"
-			set -g fish_key_bindings fish_default_key_bindings # This triggers the handler, which calls us again and ensures the user_key_bindings are executed
+			# Allow the user to set the variable universally
+			set -q fish_key_bindings; or set -g fish_key_bindings
```

Comment:
My idea was that we mention it when a variable _can't_ be universal, i.e. that the default is that it can. But then we'd need to check which can't - $PWD is one, I guess.